examples/ldap-search-entry/ldap_search_entry.bal

```diff
@@ -0,0 +1,20 @@
+import ballerina/io;
+import ballerina/ldap;
+
+public function main() returns error? {
+    // Initializes a new LDAP client.
+    ldap:Client ldapClient = check new ({
```

Comment:
Does LDAP client `init` supports included record params (IMO it should) ? If that is the case lets use that approach here. That is the preferred way when we init clients.